time/src/main/kotlin/io/spine/time/DurationExtensions.kt

```diff
@@ -0,0 +1,175 @@
+ */
+
+@file:JvmName("DurationExtensions")
+
+package io.spine.time
+
+import com.google.protobuf.Duration
+import com.google.protobuf.util.Durations.compare
+import com.google.protobuf.util.Durations.isNegative
+import com.google.protobuf.util.Durations.isValid
+import com.google.protobuf.util.Durations.toDays
+import com.google.protobuf.util.Durations.toHours
+import com.google.protobuf.util.Durations.toMicros
+import com.google.protobuf.util.Durations.toMillis
+import com.google.protobuf.util.Durations.toMinutes
+import com.google.protobuf.util.Durations.toNanos
+import com.google.protobuf.util.Durations.toSeconds
+import com.google.protobuf.util.Durations.toString
+import io.spine.protobuf.Durations2
+import io.spine.protobuf.Durations2.add
+import io.spine.protobuf.Durations2.isPositive
+import io.spine.protobuf.Durations2.isPositiveOrZero
+import io.spine.protobuf.Durations2.isZero
+import io.spine.protobuf.Durations2.toJavaTime
+
+/**
+ * Compares this duration with the passed one.
+ *
+ * @return the value `0` if `x == y`}`; a value less than `0`}` if `x < y`;
+ *     and a value greater than `0` if `x > y`.
+ */
+public operator fun Duration.compareTo(other: Duration): Int = compare(this, other)
+
+/**
+ * Returns true if this duration is valid.
+ *
+ * The [seconds][Duration.getSeconds] value must be in the range
+ * [-315,576,000,000, +315,576,000,000].
+ *
+ * The [nanos][Duration.getNanos] value must be in the range [-999,999,999, +999,999,999].
+ *
+ * **Note:** Durations less than one second are represented with a zero
+ * `seconds` field and a positive or negative `nanos` field. For durations of one second or more,
+ * a non-zero value for the `nanos` field must be of the same sign as the `seconds` field.
+ */
+public fun Duration.isValid(): Boolean = isValid(this)
+
+/**
+ * Tells whether this duration is negative or not.
+ */
+public fun Duration.isNegative(): Boolean = isNegative(this)
+
+/**
+ * Tells whether this duration is positive or not.
+ *
+ * @returns `true` if the passed value is greater than zero, `false` otherwise.
+ */
+public fun Duration.isPositive(): Boolean = isPositive(this)
+
+/**
+ * Tells whether this duration is not negative.
+ *
+ * @returns `true` of the passed value is greater or equal zero, `false` otherwise.
+ */
+public fun Duration.isPositiveOrZero(): Boolean = isPositiveOrZero(this)
+
+/**
+ * Returns `true` if the passed value is zero, `false` otherwise.
+ *
+ * @see Durations2.ZERO
+ */
+public fun Duration.isZero(): Boolean = isZero(this)
+
+/**
+ * Convert Duration to string format.
+ *
+ * The string format will contains 3, 6, or 9 fractional digits depending on the precision required
+ * to represent the exact `Duration` value. For example: "1s", "1.010s", "1.000000100s", "-3.100s".
+ * The range that can be represented by `Duration` is from -315,576,000,000 to +315,576,000,000
+ * inclusive (in seconds).
+ *
+ * @return The string representation of the given duration.
```

Comment:
Do we put periods at the end of `@return` and `@throws` in Kotlin? If not, please remove.

Reply:
From what I see in `kotlin.collections` classes, they do put periods in the end of sentences, which I find a more correct from the language point of view.